dev/tests/integration/testsuite/Magento/CatalogUrlRewrite/Model/Product/AnchorUrlRewriteGeneratorTest.php

```diff
@@ -50,32 +44,49 @@ public function setUp()
 
         $this->objectManager = Bootstrap::getObjectManager();
         $this->productRepository = $this->objectManager->create(ProductRepositoryInterface::class);
-        $this->collectionCategory = $this->objectManager->create(Category::class);
         $this->objectRegistryFactory = $this->objectManager->create(ObjectRegistryFactory::class);
     }
 
     /**
      * Verify correct generate of the relative "StoreId"
      *
+     * @param string $expect
+     * @return void
+     * @throws \Magento\Framework\Exception\LocalizedException
+     * @throws \Magento\Framework\Exception\NoSuchEntityException
      * @magentoDataFixture Magento/CatalogUrlRewrite/_files/product_with_stores.php
      * @magentoDbIsolation disabled
-     *
-     * @return void
+     * @dataProvider getConfigGenerate
      */
-    public function testGenerate(): void
+    public function testGenerate(string $expect): void
```

Comment:
For what input should the result change?
I see expectation attribute, but there's not context attribute.